src/zui/ZUIBreadcrumbs/index.tsx

```diff
@@ -0,0 +1,236 @@
+import { makeStyles } from '@mui/styles';
+import NextLink from 'next/link';
+import {
+  Box,
+  Link,
+  Menu,
+  SvgIconTypeMap,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { FC, ReactNode, useState } from 'react';
+import {
+  MoreHoriz,
+  SubdirectoryArrowLeft,
+  SubdirectoryArrowRight,
+} from '@mui/icons-material';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+
+import { Msg } from 'core/i18n';
+import messageIds from 'zui/l10n/messageIds';
+
+const useStyles = makeStyles((theme) => ({
+  icon: {
+    color: theme.palette.grey[200],
+  },
+  leftSide: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.grey[50],
+    borderRadius: '2em 0 0 2em',
+    borderRight: `1px solid ${theme.palette.grey[300]}`,
+    display: 'flex',
+    height: '26px',
+  },
+  rightSide: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.grey[50],
+    borderRadius: '0 2em 2em 0',
+    cursor: 'pointer',
+    display: 'flex',
+    height: '26px',
+  },
+}));
+
+export interface BreadcrumbTreeItem {
+  href: string;
+  icon?: OverridableComponent<SvgIconTypeMap<Record<string, unknown>, 'svg'>>;
+  id: number;
```

Comment:
Where should the value for this come from? Breadcrumbs refer to pages, and I struggle to imagine what "ID" means for a page. What is the ID of the "projects" page? What is the ID of an event page (it can't be the event ID because that same number can also be used for something else, e.g. a survey).

I think `href` should be identifier enough, so I don't think there should be a need for `id`. Correct me if you think I'm mistaken!

Reply:
I think using href should work! doing it this way is probably just because i based it on existing code in the organization tree, and did not think deeply about whether a breadcrumb would have a numeric id or not.